### PR TITLE
Abstract Debuggers fixes

### PR DESCRIPTION
**Pull request by Shadowrs**

### CODE REVIEW

File: src/main/java/org/rev317/min/debug/DGroundItems.java [hunk 1]
```diff
@@ -1,31 +1,49 @@
...
 public class DGroundItems extends AbstractDebugger {
 
+    private boolean enabled;
+    private long lastCheck = System.currentTimeMillis();
+    private int cached;
+
     @Override
     public void paint(Graphics g) {
-
+        if (System.currentTimeMillis() - lastCheck > 1000L) {
```
Comment (JKetelaar) [inline src/main/java/org/rev317/min/debug/DGroundItems.java]: Paint will only be reloaded every second anyways, why is this required?

Comment (Shadowrs) [reply src/main/java/org/rev317/min/debug/DGroundItems.java]: I wasn't aware of the tick rate paint() was called so the 1000L was just a precaution against calling `Scene.getObjects()` and generating a 2k size array every few ms. Can remove if not needed

Comment (JKetelaar) [reply src/main/java/org/rev317/min/debug/DGroundItems.java]: Would be great if it was removed, that way we can control the speed from within the abstract layer instead of every class

Comment (Shadowrs) [reply src/main/java/org/rev317/min/debug/DGroundItems.java]: Done
